### ISSUE TITLE
Jet Smart Filter Tooltip Label

### ISSUE THREAD

**Issue by stasonua0** [CrocoBlock/suggestions#4513]
Good evening! Consider adding functionality to Jet Smart Filters to display helper Tooltips when hovering over a filter label as in https://prnt.sc/22ubt39.

/*Оригинал*/
Добрый вечер! Рассмотрите возможность добавить функционал в Jet Smart Filters для вывода вспомогательных Tooltip при наведении на метку фильтра как на примере https://prnt.sc/22ubt39.

**Comment by nreljed** [comment]
it's a good idea that I came across while actively looking for filter plugins.
unfortunately for jetsmartfilters a new challenger is entering the arena.

[FilterEverything](https://demo.filtereverything.pro/product-category/electronics/smartphones/)

now we have gone beyond the stage of adding functions because a standard has already been established and we must now catch up and align ourselves with hot issues such as

- SEO url
- Ajax loading statuts 
- dedicated Widget show results (plural and singular)
- Shortcodes
- everything that I have not noted.

to go further just out of pure curiosity! I wanted to understand who is behind these plugins and what structure they have to make their plugins, the rate of their updates and reactivity.
the result surprised me

- Filter Everything (it's only one person)
- Facetwp (it's 2 people)
- Gridbuilder (it's 1 person)
- Jetsmartfilters from what I understood only one person shared for several projects ??

the quality of the plugins is very different and none has a **real dedicated team** as you might think.

this may be the answer to why jetsmartfilters cannot match the competition at the moment (2021).
perhaps for 2022 the management at crocoblock will change its strategy?

**Comment by Lonsdale201** [comment]
If you are using the Jettricks you can do this.

**Comment by stasonua0** [comment]
> 

You are wrong, for the labels of the filter itself it is impossible to put the tooltip from the Jet Tricks plugin

**Comment by Lonsdale201** [comment]
> > 
> 
> You are wrong, for the labels of the filter itself it is impossible to put the tooltip from the Jet Tricks plugin

Mine is working fine, so idk, you why cant do this.
https://prnt.sc/240iq0w - just use the .jet-filter-label custom selector for the target :)

**Comment by nreljed** [comment]
@Lonsdale201  the problem is a very heavy action to have to install a plugin for a simple tooltip it is a noble cause to think of this as a solution... but not ideal in practice especially in a **context very oriented towards the centralization of functions and good practices** specially designed for search filters. 

FilterEverything proves to us that a lighter and more elegant solution exists why deny this already established fact?

jetsmartfilters still have many challenges to overcome

I take this opportunity to relaunch the need for [formatting for the numbers in jetsmartfilters range slider](https://github.com/Crocoblock/suggestions/issues/4524)

`conceptual illustration made with adobe xd for illustration of 4
different use of range price + tooltip option`
![option](https://user-images.githubusercontent.com/82481592/147121177-ee2a7e76-6e13-4e8d-9956-8d2a31dc63c2.jpg)


addition added to #2998

**Comment by stasonua0** [comment]
> > > 
> > 
> > 
> > You are wrong, for the labels of the filter itself it is impossible to put the tooltip from the Jet Tricks plugin
> 
> Mine is working fine, so idk, you why cant do this. https://prnt.sc/240iq0w - just use the .jet-filter-label custom selector for the target :)

I have a lot of filters and some can be hidden and others shown depending on the category. Still confident that Jet Tricks can solve this problem? https://prnt.sc/24buyg8

**Comment by Lonsdale201** [comment]
> @Lonsdale201 the problem is a very heavy action to have to install a plugin for a simple tooltip it is a noble cause to think of this as a solution... but not ideal in practice especially in a **context very oriented towards the centralization of functions and good practices** specially designed for search filters.
> 
> FilterEverything proves to us that a lighter and more elegant solution exists why deny this already established fact?
> 
> jetsmartfilters still have many challenges to overcome
> 
> I take this opportunity to relaunch the need for [formatting for the numbers in jetsmartfilters range slider](https://github.com/Crocoblock/suggestions/issues/4524)
> 
> `conceptual illustration made with adobe xd for illustration of 4 different use of range price + tooltip option` ![option](https://user-images.githubusercontent.com/82481592/147121177-ee2a7e76-6e13-4e8d-9956-8d2a31dc63c2.jpg)
> 
> addition added to #2998


You are absolutely right. The idea is good anyway, I'd rather just say it can be done in-house, but a few lines of css can be done very easily.

**Comment by stasonua0** [comment]
I decided to use the following scheme:
Installed the "Tooltip CK" plugin
Added to the tags of filters https://prnt.sc/253i0z4
As a result, we have https://prnt.sc/253i3hz
